lib/spack/spack/config.py

```diff
@@ -1098,6 +1102,75 @@ def ensure_latest_format_fn(section):
     return update_fn
 
 
+#: default installation root, relative to the Spack install path
+default_install_root = os.path.join(spack.paths.opt_path, 'spack')
+
+
+def parse_install_tree():
```

Comment:
Is there something preventing this from living in `store.py`? I see the bulk of this logic was moved from there. The `config.py` module doesn't contain code specific to processing specific sections (e.g. `compilers`).

Reply:
I moved it because I think it is better separation of concerns to have it there, but I can move it back to `store.py`.

In the long run we might want to create an entire module for indirections of the config system for backwards compatibility. The more we change configs the more those will accumulate over time.